### ISSUE TITLE
inference

### ISSUE THREAD

**Issue by lijain** [thuanz123/realfill#1]
You guys are doing a great job, and I have a few questions for you:

1 What is the effect of outpaint without lora training?

2 Have you tested the maximum times of outpaint original?

3 There is no direct inference script

**Comment by thuanz123** [comment]
1. Full finetuning the model like DreamBooth will make the model completely overfit to the reference images, LoRA and some additional small tricks should be used to avoid that.

2. I dont understand this question

3. Just slab the original inference script of diffusers is fine, but I will add it in this repo

**Comment by xinli2008** [comment]
hello, another question to bother you. you set the validation in the code, but the output validation images seems not to be saved.

**Comment by thuanz123** [comment]
The images are already saved in tensorboard where you can view both the loss, images and some other hyper-param. I find it more convenient to visualize this way.

**Comment by xinli2008** [comment]
thank you!  can see the 4 validation images in the tensorboard.
